prefs.js

```diff
@@ -227,7 +227,6 @@ var Settings = GObject.registerClass({
         // Set a reasonable initial window height
         this.widget.connect('realize', () => {
             const rootWindow = this.widget.get_root();
-            rootWindow.set_size_request(-1, 850);
```

Comment:
Please use instead:

```js
if (!Me)
    rootWindow.set_size_request(-1, 850);
```

So that it will still have some reasonable size (can be even smaller) when running as standalone window (i.e. just running `gjs prefs.js` for development purposes).

Reply:
Not quite that syntax because 850 is still too big for some displays.

Reply:
That's just for the debugging case, were we need to provide a default window size... Can be smaller but we still need to have one.